### ISSUE TITLE
User Profile Features

### ISSUE THREAD

**Issue by vektor8891** [zsebtanar/zsebtanar-proto#588]
In the profile tab, the user should be able to:
- [ ] change their name
- [ ] change their email address
- [ ] change their password
- [ ] delete profile (button is already there)

Additional comments:
- [ ] date of registration do not need to be displayed (it is not important for the user but for us)
- [ ] if a user used OAuth to sign up, they should be able to create a password and use that to log in

**Comment by maxinteger** [comment]
@vektor8891 
> if a user used OAuth to sign up, they should be able to create a password and use that to log in

Why is this needed? I never see this option in any web app.

**Comment by vektor8891** [comment]
@maxinteger Agreed, this feature is not too common. We can drop this from the list.
